iree/test/e2e/regression/BUILD

```diff
@@ -64,11 +65,10 @@ iree_lit_test_suite(
 )
 
 iree_check_single_backend_test_suite(
-    name = "check_regression_dylib-llvm-aot",
+    name = "check_regression_lowering_config_dylib-llvm-aot",
     srcs = [
-        "linalg_ext_ops.mlir",
         "lowering_config.mlir",
-    ] + BACKEND_TESTS,
```

Comment:
so the backend tests are not ran anymore for dylib target?

Reply:
Good catch! I meant to split the lowering_config one out. Let me fix that. THanks!